src/gurt/SConscript

```diff
@@ -0,0 +1,51 @@
+#!python
+# (C) Copyright 2020 Intel Corporation.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+# The Government's rights to use, modify, reproduce, release, perform, display,
+# or disclose this software are subject to the terms of the Apache License as
+# provided in Contract No. 8F-30005.
+# Any reproduction of computer software, computer software documentation, or
+# portions thereof marked with this legend must also reproduce the markings.
+#
+"""Build libgurt"""
+
+SRC = ['debug.c', 'dlog.c', 'hash.c', 'misc.c', 'heap.c', 'errno.c',
+       'fault_inject.c']
+
+def scons():
+    """Scons function"""
+
+    Import('env', 'prereqs', 'CART_VERSION')
+
+    env.AppendUnique(LIBPATH=[Dir('.')])
+
+    denv = env.Clone()
+
+    denv.AppendUnique(CPPPATH=[Dir('.').srcnode()])
+    denv.AppendUnique(LIBS=['pthread', 'yaml'])
+    prereqs.require(denv, 'uuid')
+
+    gurt_targets = denv.SharedObject(SRC)
+    gurt_lib = denv.SharedLibrary('libgurt', gurt_targets,
+                                  SHLIBVERSION=CART_VERSION)
+    denv.InstallVersionedLib('$PREFIX/lib64/', gurt_lib,
+                             SHLIBVERSION=CART_VERSION)
+
+    Default(gurt_lib)
+    Export('gurt_lib', 'gurt_targets')
+
+if __name__ == "SCons.Script":
+    scons()
```

Comment:
(pylint-missing-final-newline) Final newline missing
(pylint-mixed-line-endings) Mixed line endings LF and CRLF